### PR TITLE
KOGITO-2360 - Enable webhooks and comment trigger on Kogito backend P…

### PR DESCRIPTION
**Pull request by MarianMacik**
…R checks

PR set:
* kiegroup/kogito-runtimes#550 
* kiegroup/kogito-apps#258
* kiegroup/kogito-examples#272

### CODE REVIEW

**Comment by MarianMacik** [comment]
@danielezonca I have included you to just confirm my fix, it seems that when you were rewriting this to pull from master, you copy-pasted probably the wrong repo:
https://github.com/kiegroup/kogito-examples/pull/269/files#diff-58231b16fdee45a03a4ee3cf94a9f2c3R43

It should be kogito-examples there.

**Review by danielezonca — APPROVED** [review]

**Comment by danielezonca** [comment]
@MarianMacik thanks for the fix, my bad :)

**Review by radtriste — APPROVED** [review]

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

File: Jenkinsfile [hunk 1]
```diff
@@ -1,5 +1,9 @@
 @Library('jenkins-pipeline-shared-libraries')_
 
+def CHANGE_AUTHOR = "$ghprbPullAuthorLogin"
+def CHANGE_BRANCH = "$ghprbSourceBranch"
+def CHANGE_TARGET = "$ghprbTargetBranch"
+
```
Comment (Ginxo) [inline Jenkinsfile:6]: I would replace them by
```
def CHANGE_AUTHOR = env.CHANGE_AUTHOR ?: ghprbPullAuthorLogin
def CHANGE_BRANCH = env.CHANGE_BRANCH ?: ghprbSourceBranch
def CHANGE_TARGET = env.CHANGE_TARGET ?: ghprbTargetBranch
```
in order to allow either multibranch pipeline job types and regular job types + hooks, wdyt?

**Review by Ginxo — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (MarianMacik) [reply Jenkinsfile:6]: Fixed. Just prioritized regular job type variables over multibranch ones.

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**